### PR TITLE
4.16.0 patch notes

### PR DESCRIPTION
**Pull request by ScottSuarez**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,31 @@
 ## 4.16.0 (Unreleased)
+FEATURES:
+* **New Data Source:** dataproc_metastore_service ([#4155](https://github.com/hashicorp/terraform-provider-google-beta/pull/4155))
```
Comment (megan07) [inline CHANGELOG.md:3]:
```suggestion
* **New Data Source:** `dataproc_metastore_service` ([#4155](https://github.com/hashicorp/terraform-provider-google-beta/pull/4155))
```

Comment (rileykarson) [inline CHANGELOG.md:3]:
```suggestion
* **New Data Source:** `google_dataproc_metastore_service` ([#4155](https://github.com/hashicorp/terraform-provider-google-beta/pull/4155))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,31 @@
 ## 4.16.0 (Unreleased)
+FEATURES:
+* **New Data Source:** `dataproc_metastore_service` ([#4155](https://github.com/hashicorp/terraform-provider-google-beta/pull/4155))
```
Comment (rileykarson) [inline CHANGELOG.md:3]:
```suggestion
* **New Data Source:** `google_dataproc_metastore_service` ([#4155](https://github.com/hashicorp/terraform-provider-google-beta/pull/4155))
```

**Review by rileykarson — APPROVED** [review]
Watch out when accepting suggestions- resolving comments and pushing changes don't generate emails (at least w/ my settings) so you often won't notify reviewers of a change when you do so! I try to leave a comment each pass, to indicate the baton's back in the other person's hands.

**Comment by ScottSuarez** [comment]
ah, makes sense. I'll try to comment when resolving changes in the future !

**Pull request merged**